internal/cri/server/container_image_mount_linux.go

```diff
@@ -0,0 +1,92 @@
+//go:build linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package server
+
+import (
+	"fmt"
+	"os"
+	"sync"
+
+	"github.com/containerd/containerd/v2/core/mount"
+	kernel "github.com/containerd/containerd/v2/pkg/kernelversion"
+)
+
+var (
+	volatileSupported     bool
+	volatileSupportedOnce sync.Once
+)
```

Comment:
nit: this can be moved to a centralized place? maybe in a followup. I guess similar logic is used somewhere else?

Reply:
We can do it when new functionality needs this.